### PR TITLE
npm audit fix

### PR DESCRIPTION
**Pull request by mjlumetta**
got some alerts, ran `npm audit fix`

### CODE REVIEW

**Comment by jasalisbury** [comment]
Has this already been merged? I don't see any files being changed

**Comment by mjlumetta** [comment]
uhhh good question @jasalisbury. i was trying to commit this change: https://github.com/department-of-veterans-affairs/developer-portal/pull/295/commits/abc012a7f576ae29b5cb35dfcc53ddd796e50025

**Comment by kalilsn** [comment]
@mjlumetta it looks like those changes were merged in https://github.com/department-of-veterans-affairs/developer-portal/pull/269. Perhaps this is the wrong branch? Or the new audit fix changes didn't get pushed to the remote?

**Comment by mjlumetta** [comment]
ah yes i re-used this branch and did not actually commit the new changes, and it's like the same commit message... smart of me, i know

**Comment by mjlumetta** [comment]
@jasalisbury @kalilsn i'm dumb it's fixed now

**Comment by kalilsn** [comment]
>ah yes i re-used this branch and did not actually commit the new changes, and it's like the same commit message... smart of me, i know

😂 you're just _too_ consistent!

File: package-lock.json [hunk 1]
```diff
@@ -8431,7 +8433,7 @@
       "dependencies": {
         "combined-stream": {
           "version": "1.0.6",
-          "resolved": "https://registry.npmjs.org/combined-stream/-/combined-stream-1.0.6.tgz",
+          "resolved": "http://registry.npmjs.org/combined-stream/-/combined-stream-1.0.6.tgz",
```
Comment (kalilsn) [inline package-lock.json]: I want to ask you to remove these unnecessary https -> http changes, but honestly if this is still happening then it's probably not worth worrying about.

Comment (mjlumetta) [reply package-lock.json]: it makes no sense to me but i also don't want to get in a fight with the automated tools, so not gonna worry about it, yeah. but i see your point

**Pull request merged**